config/foreman.migrations/20160831180000_foreman_proxy_tftp_grub2.rb

```diff
@@ -0,0 +1,5 @@
+root = answers['foreman_proxy']['tftp_root']
+dirs = answers['foreman_proxy']['tftp_dirs']
+dirs << "#{root}/grub"
```

Comment:
The first run will be using the default answers file which doesn't contain much: https://github.com/theforeman/foreman-installer/blob/develop/config/foreman-answers.yaml#L14